Crane/app/src/main/java/androidx/compose/samples/crane/details/DetailsActivity.kt

```diff
@@ -193,12 +189,14 @@ fun CityMapView(
         LatLng(latitude.toDouble(), longitude.toDouble())
     }
 
-    // Set our initial map position when the city location changes
-    LaunchedEffect(cityLocation) {
-        cameraPositionState.position = CameraPosition.fromLatLngZoom(
-            cityLocation,
-            InitialZoom
-        )
+    // Set our initial camera position whenever the city location changes.
+    remember(cityLocation) {
```

Comment:
I don't think this is a good usage of `remember` TBH. Maybe ask eng? If you don't want it to be in a `LaunchedEffect` because setting the position is delayed one frame, it'd probably be best to have it as it was before: being called on every recomposition.

Reply:
Yeah I don't like it either, I think you are right. The single frame for `LaunchedEffect` doesn't really matter at the end of the day so I have switched it to that.